management/prompt.go

```diff
@@ -33,11 +36,73 @@ var allowedPromptsWithPartials = []PromptType{
 	PromptLogin,
 	PromptLoginID,
 	PromptLoginPassword,
+	PromptLoginPasswordLess,
 }
 
 // PromptType defines the prompt that we are managing.
 type PromptType string
 
+// ScreenName is a type that represents the name of a screen.
+type ScreenName string
+
+// InsertionPoint is a type that represents the insertion point of a screen.
+type InsertionPoint string
+
+const (
+	// ScreenLogin represents the signup screen.
```

Comment:
I'm not sure, but maybe this should be the `login` screen.

Signup screen is below:
ScreenSignup ScreenName = "signup"